### PR TITLE
Fixed the right overflow of small cards in the cards section. 

### PR DESCRIPTION
**Pull request by jaivsh**
### Pull Request Template

**Description:**

I've modified the card-design in such a way such that it doesn't overflow on any screen size, ranging from a large sized tablet to a small sized smartphone. 

**Changes Made:**

-Modified the design of the card so as to allow itself to be more accomodative while adhering to the basic layout of all the cards.
- Changed the width and height of affected cards to prevent the overflow.

**Related Issue:**

-#196 

**Screenshots/GIFs:**

ss1

**Testing Done:**

I've tested the modification in the following emulators in android studio: 
- Pixel 4
- Nexus 10
- Pixel XL

**Checklist:**

- [X] The code follows the project's coding style guidelines.
- [X] Unit tests have been added or updated to cover the changes.
- [X] Documentation, if applicable, has been updated to reflect the changes.
- [X] All new and existing tests pass successfully.

Please review and merge this PR at your earliest convenience.

Thank you!

### CODE REVIEW

File: lib/ui_components/cards/All Cards/blog_card_dark_mode/card_6.dart [hunk 1]
```diff
@@ -11,11 +11,11 @@ class SixthCard extends StatelessWidget {
...
             color: Colors.black,
             padding: const EdgeInsets.only(left: 5, top: 2, bottom: 2),
-            height: (MediaQuery.of(context).size.width / 2.4) * 0.83,
+            height: (MediaQuery.of(context).size.width / 2.4) * 0.95,
             width: MediaQuery.of(context).size.width -
                 MediaQuery.of(context).size.width / 2.4 -
-                8,
-            child: const Column(
+                32,
+            child: Column(
```
Comment (ashdude1401) [inline lib/ui_components/cards/All Cards/blog_card_dark_mode/card_6.dart:18]: Use const before Column as there is not dynamic data associated with it .

File: lib/ui_components/cards/All Cards/blog_card_dark_mode/card_6.dart [hunk 2]
```diff
@@ -11,11 +11,11 @@ class SixthCard extends StatelessWidget {
           Container(
             color: Colors.black,
             padding: const EdgeInsets.only(left: 5, top: 2, bottom: 2),
-            height: (MediaQuery.of(context).size.width / 2.4) * 0.83,
+            height: (MediaQuery.of(context).size.width / 2.4) * 0.95,
```
Comment (ashdude1401) [inline lib/ui_components/cards/All Cards/blog_card_dark_mode/card_6.dart:14]: The scren is still overflowing in my phone check.

Comment (ashdude1401) [inline lib/ui_components/cards/All Cards/blog_card_dark_mode/card_6.dart:14]: The cards are still overflowing in my phone . Please fix it .

Comment (ashdude1401) [reply lib/ui_components/cards/All Cards/blog_card_dark_mode/card_6.dart:14]: image
image
image
image

Comment (ashdude1401) [reply lib/ui_components/cards/All Cards/blog_card_dark_mode/card_6.dart:14]: After fixing these let me know

Comment (jaivsh) [reply lib/ui_components/cards/All Cards/blog_card_dark_mode/card_6.dart]: @ashdude1401 **Sir, I've implemented the changed for each specific card. Have confirmed it by running the app on smaller sized emulators. Please have a review!!**

Also, I've implemented the const wherever possible. However, im using the latest version of flutter, and as per it, there have been modifications made by the dart and flutter in its new versions, which does not allow const in specific places, like in line 17 above as you mentioned. Thanks!


Screenshot_20230526_110707
Screenshot_20230526_110714
Screenshot_20230526_110721
Screenshot_20230526_110659
Screenshot_20230526_110648
Screenshot_20230526_104308

File: lib/ui_components/cards/All Cards/blog_card_dark_mode/card_6.dart (hunk 1 above)
Comment (jaivsh) [reply lib/ui_components/cards/All Cards/blog_card_dark_mode/card_6.dart]: Hello Sir, I've modified the PR and implemented the const wherever possible. However, im using the latest version of flutter, and as per it, there have been modifications made by the dart and flutter in its new versions, which does not allow const in specific places, like in line 17-18 above as you mentioned. Thank you!

File: lib/ui_components/cards/All Cards/blog_card_dark_mode/card_6.dart (hunk 2 above)
Comment (ashdude1401) [reply lib/ui_components/cards/All Cards/blog_card_dark_mode/card_6.dart]: @jaivsh I have checked again it is still overflowing in phone, you have make make it responsive for all the screen .All phone screen are not covered by emulator I think mine is one of them .My screen screen size is 6.8 inch. Some screen you have to handle separately and secondly , I on the latest version of flutter and dart , it is still showing me the suggestion of adding const at line 17 .So, please these issue again and let me know once you are done.

**Comment by Debajyoti14** [comment]
@jaivsh There are a lot of merge conflicts due to the merge of a similar PR. Can you make a new commit to fix all these conflicts?

**Pull request closed without merging**